src/MultiSharedMultipleRegions/MSMRmain.cpp

```diff
@@ -27,29 +27,29 @@ int main(int argc, char *argv[] ) {
   // Multiple regions related to --slow that take different amounts of time.
 
   if (Slow) {
-    fpcsc::sleep_for_secs(5);
+    fpcsc::busy_sleep_for_secs(5);
   } else {
-    fpcsc::sleep_for_secs(3);
+    fpcsc::busy_sleep_for_secs(3);
```

Comment:
I like the idea of having a busy_sleep version that perf can detect. It just conflicts a bit with debugability. Would you be so nice an maybe add an abstraction where we can, with a compile switch, use busy VS normal sleeping?